### ISSUE TITLE
Add more GHC version tests

### ISSUE THREAD

**Issue by ndmitchell** [digital-asset/ghcide#26]
We know it works with GHC 8.4 today, but without CI, that's likely to stop being true relatively quickly.

Probably also want a GHC 8.8 test too.

**Comment by cocreature** [comment]
PRs welcome :slightly_smiling_face:

**Comment by ndmitchell** [comment]
PR in https://github.com/digital-asset/ghcide/pull/29.

**Comment by cocreature** [comment]
We now have CI for 8.4. I believe `lsp-test` is blocked on something for `8.8` atm but I don’t recall the details.

**Comment by ndmitchell** [comment]
Close this an open a separate ticket for 8.8 support? There are plenty of things that are blocked, so I don't suggest trying to hard for a few weeks.

**Comment by cocreature** [comment]
Closing in favor of #33
